### PR TITLE
Allow MQTTS and WSS for MQTT Integration

### PR DESCRIPTION
**Pull request by mpfeil**
## Description
This PR updates all `grpc` packages to the latest versions and adds a Github Action to build and push the Docker Image to the Github Registry (`ghcr`).
Also this PR includes changes so that `mqtts` and `wss` are supported by the `mqtt-osem-integration`.

## Motivation and Context

Allow `mqtts` and `wss` connections within the `mqtt-osem-integration`. https://github.com/sensebox/mqtt-osem-integration/issues/4

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code has been linted using `yarn run lint`.
- [x] My code does not break the tests (`yarn run test`)
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: .github/workflows/registry.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: "Docker: Github Registry"
+
+on:
+  push:
+    branches: [ feat-mqtts ]
```
Comment (ubergesundheit) [inline .github/workflows/registry.yaml:5]: Don't forget to update before merge ;)

**Comment by ubergesundheit** [comment]
Mh any idea why the other workflows (test, lint, ..) are not running? This looks good, but green tests should be there.

**Comment by mpfeil** [comment]
Mh good question. Maybe because it is still WIP?

**Comment by ubergesundheit** [comment]
Mh it should run for `pull_request` against `master` branch anyway https://github.com/sensebox/openSenseMap-API/blob/master/.github/workflows/test.yaml#L3-L9

**Comment by mpfeil** [comment]
Updated docs can be found here: https://github.com/sensebox/docs.sensebox.de/pull/68

**Pull request merged**